Docs: Update Grafana Teams documentation (#86942)
grafana/grafana: docs/sources/administration/team-management/_index.md

Reviewer: You'll need to run `prettier` on this file

---

> aliases:
>   -

Reviewer: ```suggestion
```

  - microservices
  - architecture
menuTitle: Grafana Teams
title: Grafana Teams
weight: 100
---

# Grafana Teams

Grafana Teams makes it easy to organize and administer groups of users in your enterprise. Teams allows you to grant permissions to a group of users instead of granting permissions to individual users one at a time.

Teams are useful in a wide variety of scenarios, such as when onboarding new colleagues or needing access to reports on secure financial data. When you add a user to a team, they get access to all resources assigned to that team.

## Teams concepts

A Grafana Team is a group of users within an organization that have common permissions, including access to dashboards and data sources, and those permissions apply to **all members** of that team. For example, instead of assigning six users access to the same dashboard, you can create a team that consists of those users and assign dashboard permissions to the team. A user can belong to multiple teams.

A Team grants permissions to a wide variety of resources including:

- dashboards
- data sources
- folders
- alerts
- reports
- cloud access policies
- annotations
- playlists

{{< admonition type="note" >}}

> All members of a Grafana Team have the same exact permissions. A single Team cannot have members with different access levels to resources shared within that Team.
> {{< /admonition >}}
> 
> A user can be a `Member` or an `Administrator` for a given Team. `Members` of a Team inherit permissions from the team, but they do not have team administrator privileges, and cannot edit the team itself. Team `Administrators` can add members to a team and update its settings, such as the team name, team member’s team roles, UI preferences, and home dashboard.

Reviewer: nit: use contractions for an informal tone (https://grafana.com/docs/writers-toolkit/write/style-guide/voice-tone-guidelines/#tone).

```suggestion
All members of a Grafana Team have the same exact permissions.
A single Team can't have members with different access levels to resources shared within that Team.
{{< /admonition >}}

A user can be a `Member` or an `Administrator` for a given Team.
`Members` of a Team inherit permissions from the team, but they don't have team administrator privileges, and can't edit the team itself.
Team `Administrators` can add members to a team and update its settings, such as the team name, team member’s team roles, UI preferences, and home dashboard.
```

---
aliases:

>   - 

Reviewer: The aliases need to be moved from https://github.com/grafana/grafana/pull/86942/files#diff-de47761a6b83e1cd74a87c05843e44b98549e5752c2234829dd174977de94ae2

> description: This document introduces Grafana Teams and concepts.

Reviewer: I think the description could be revised to better suit that it is presented to the user as part of search engine results.

Author: @jdbaldry I am not sure how you'd like to re-word the description, it's in line with most other descriptions in this repo.

Reviewer: It's probably OK, but for the future, keep in mind the guidance in https://grafana.com/docs/writers-toolkit/write/front-matter/#description-required.
